cinthia_argote/tic_tac_toe/src/PlayerTest.java

```diff
@@ -0,0 +1,20 @@
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class PlayerTest {
+  private Player player;
+  
+  @Before
+  public void setUp(){
+    player = new Player("name",'x');
+  }
+  
+  @Test
+  public void setAttempts(){
+    player.setAttempts(2);
+    int attempts = player.getAttempts();
+    assertEquals(2, attempts);
+  }
```

Comment:
I think this test is not worth it. It seems you are testing an access modifier. I would suggest testing the logic that you implement as you did in BoardTest.